website/content/docs/platform/k8s/vso/sources/vault/client-cache.mdx

```diff
@@ -0,0 +1,266 @@
+---
+layout: docs
+page_title: Enable the encrypted Vault client cache
+description: >-
+  Learn how to enable the operator's encrypted Vault Client cache for dynamic secrets
+---
+
+# Enable the encrypted Vault client cache
+
+The Vault Secrets Operator supports storing and encrypting its Vault client cache using Vault Server's [transit secrets engine](/vault/docs/secrets/transit). By default client cache storage is not enabled, since it requires extra Vault Server setup, but this guide will walk you through the steps to enable and configure it.
```

Comment:
I think the concept of what the client cache is itself probably deserves a bit more introduction:

```suggestion
The Vault Secrets Operator can optionally cache Vault client information such as Vault tokens and leases in Kubernetes Secrets within its own namespace. The client cache enables seamless upgrades because Vault tokens and dynamic secret leases can continue to be tracked and renewed through leadership changes. By default, the client cache is not encrypted because it requires extra configuration and Vault Server setup. But the Vault Secrets Operator supports encrypting the client cache using Vault Server's [transit secrets engine](/vault/docs/secrets/transit). This guide will walk you through the steps to enable and configure client cache encryption.
```

Reply:
So I tried putting this expanded info on the [main Vault source index page](https://vault-23tw59zt8-hashicorp.vercel.app/vault/docs/platform/k8s/vso/sources/vault#vault-client-cache). Let me know what y'all think.